### ISSUE TITLE
Reduce usage of <br/>

### ISSUE THREAD

**Issue by magnetic-eagle** [flightgear-polska/flightgear-polska.github.io#2]
Instead of using many `<br/>` tags in a row, and that repeatedly, instead add some `padding` in the CSS stylesheet. It makes the HTML file much less cluttered ! ;)

**Comment by SP-NTX** [comment]
i can do but i have more to do now cause my friends dont like design

**Comment by magnetic-eagle** [comment]
Yeah, when you do the spacing by CSS padding it's much eaier and faster to change the design later ! ;) If you need any help with this project, just open an issue - it'll appear in my GitHub feed, and I'll most probably be able to help you ! (I did quite a bit of HTML+CSS+JS) coding a while ago …)
